File: app/src/main/java/com/example/keeper/fragments/SettingsFragment.java
package com.example.keeper.fragments;

import android.os.Bundle;

import android.preference.PreferenceFragment;

import com.example.keeper.R;

public class SettingsFragment extends PreferenceFragment {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.preferences);
    }

}

File: app/src/main/java/com/example/keeper/fragments/SettingsFragment.kt
package com.example.keeper.fragments

import android.os.Bundle
import android.support.v7.preference.PreferenceFragmentCompat
import com.example.keeper.R

class SettingsFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(bundle: Bundle?, s: String?) {
        addPreferencesFromResource(R.xml.preferences)
    }
}